doc/src/modules/vector/api/classes.rst

```diff
@@ -2,13 +2,13 @@
 Essential Classes in sympy.vector (doctrings)
 =============================================
 
-CoordSysCartesian
-=================
+CoordSystem3D
```

Comment:
+1. We need to think about the class structure for the coordinate systems classes a little better. What I had in mind when I designed it was to have a CoordSys base class, which defines the skeleton features of any coordinate system in sympy.vector. If needed, as @Upabjojr suggests, we could even support N-dimensions. Every type of coordinate system would then specialize the base class' properties and define their own features.